spec/numberformat.html

```diff
@@ -347,6 +406,12 @@ <h1>Intl.NumberFormat.prototype.resolvedOptions ( )</h1>
           1. Let _v_ be the value of _nf_'s internal slot whose name is the Internal Slot value of the current row.
           1. If _v_ is not *undefined*, then
             1. Perform ! CreateDataPropertyOrThrow(_options_, _p_, _v_).
+        1. If _nf_.[[RoundingType]] is ~morePrecision~, then
+          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"morePrecision"*).
+        1. Else if _nf_.[[RoundingType]] is ~lessPrecision~, then
+          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"lessPrecision"*).
+        1. Else,
+          1. Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"auto"*).
```

Comment:
Would it make more sense to just introduce a [[RoundingPriority]] slot and avoid the potential of this recovery logic falling out of sync in the future (and also the minor weirdness of "roundingPriority" being required to appear as the last property of the object returned from `resolvedOptions()`)?

Reply:
Would rather not make this big of a change at this point. Also, this logic reflects that RoundingPriority is now a computed property: if compact notation is used, it gets computed as "auto", which is needed for round-tripping.

Reply:
@ryzokuken Thoughts on the above? I can live with it, albeit unhappily.